### DISCUSSION TITLE
hi, I want to make sure that the map in itwin.js is also loaded with tiles, right?

### DISCUSSION THREAD

**Discussion by shenshengyi** [iTwin/itwinjs-core#1204] [Q&A] [answered]
hi, I want to make sure that the map in itwin.js is the same as the Geometry model and the reality Data. They are all loaded using tiles, right?Thanks.

**Comment by pmconne** [comment, marked as answer]
> I want to make sure that the map in itwin.js is the same as the Geometry model and the reality Data.

The same **what**?

**Reply by shenshengyi** [reply]
> 
> 
> > I want to make sure that the map in itwin.js is the same as the Geometry model and the reality Data.
> 
> The same **what**?

I just want to make sure that the map in itwin.js is displayed using loading tiles? Thanks.

**Reply by pmconne** [reply]
Yes. With the exception of decorations, all graphics in iTwin.js are displayed using tiles.

**Reply by shenshengyi** [reply]
ok, thank you very much .
